# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1217874818 14400
#      Mon Aug 04 14:33:38 2008 -0400
# Node ID 4363bc94171ac18fa0ce5ccf6a7550712b9c1ca4
# Parent  997d4384cec43cd93ee39d272c809e4bdcf37133
aclocal.m4 (OCTAVE_PROG_SED): fail if no usable sed is found

diff --git a/ChangeLog b/ChangeLog
--- a/ChangeLog
+++ b/ChangeLog
@@ -1,10 +1,12 @@
 2008-08-04  John W. Eaton  <jwe@octave.org>
 
+	* aclocal.m4: Fail if no usable version of sed is found.
+
 	* Makefile.in (header-msg): Mention GNU Make requirement.
 	Update GCC version info.
 
 2008-07-31  John W. Eaton  <jwe@octave.org>
 
 	* octMakefile.in (DISTFILES): Add "missing" to the list.
 
 2008-07-30  John W. Eaton  <jwe@octave.org>
diff --git a/aclocal.m4 b/aclocal.m4
--- a/aclocal.m4
+++ b/aclocal.m4
@@ -794,16 +794,19 @@ if test -z "$SED"; then
 	  _max=$_count
 	  octave_cv_path_sed=$_sed
 	fi
       done
     done
     rm -rf "$tmp"
   ])
   SED=$octave_cv_path_sed
+  if test -z "$SED"; then
+    AC_MSG_ERROR([no usable version of sed found])
+  fi
 fi
 AC_SUBST(SED)
 AC_MSG_RESULT([$SED])
 ])
 dnl
 dnl Find Perl.
 dnl
 dnl OCTAVE_PROG_PERL
